eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/ISBNConverter.php

```diff
@@ -82,7 +82,7 @@ public function toFieldDefinition(StorageFieldDefinition $storageDef, FieldDefin
     {
         $fieldDef->fieldTypeConstraints->fieldSettings = new FieldSettings(
             array(
-                'isISBN13' => !empty($storageDef->dataInt1) ? (bool)$storageDef->dataInt1 : true,
+                'isISBN13' => !empty($storageDef->dataInt1),
```

Comment:
Well, I guess it's very subjective. I don't like using ternary operator for result that already was determined by its condition. However I get the point about readability.

@tomaszmadeyski: @andrerom has the final say, so I'm sorry I made you change your original approach just to go back to it :) 
Please don't forget though that if `dataInt1` is `null`, it should return actually `false` (whole point of this PR) :-)